gradle.properties

```diff
@@ -4,5 +4,6 @@ ktlintVersion = 0.48.2
 #default values - will be overwritten by ~/.gradle/gradle.properties
 githubUser = ""
 githubToken = ""
+org.gradle.jvmargs=-Xmx8G
```

Comment:
Wieso war das notwendig?

Reply:
this is mentioned in the onboarding steps.
Should the build fail due to not enough memory, try to add org.gradle.jvmargs=-Xmx8G to your gradle.properties file

i think it got pushed with the other stuff. should I remove it and keep it only locally?

Reply:
Ja bitte entferne das erstmal von remote. Wir können das mal in der retro besprechen.
Wenn andere auch Probleme mit zu wenig Memory haben, dann können wir es ja nach remote pushen.

Reply:
done